src/main/webapp/app/entities/companion-diagnostic-device/companion-diagnostic-device-update.tsx

```diff
@@ -47,7 +47,7 @@ export const CompanionDiagnosticDeviceUpdate = (props: ICompanionDiagnosticDevic
     for (const stv of specimenTypeValues) {
       if (!WHOLE_NUMBER_REGEX.test(stv.id)) {
         try {
-          const result = await props.createSpecimenType({ type: stv.id, name: stv.id });
+          const result = await props.createSpecimenType({ type: stv.id, name: stv.id, companionDiagnosticDevices: [] });
```

Comment:
When creating entity, if it is not associated with other entity, should this stay as null, instead of making a list?

Reply:
I'll make it null.